Lab3.ipynb

```diff
@@ -0,0 +1,401 @@
+    "$$\\begin{equation*}\n",
+    "\\int_{x}^{\\infty} pdf(x) \\,dx\n",
+    "\\end{equation*}$$\n",
+    "Where we are looking for x."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The distance the asteroid would need to move is 5.489676406965609 arcseconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob5sig = 1-1/(3.5e6)\n",
+    "\n",
+    "dist3 = stats.rayleigh.ppf(prob5sig, scale=1)\n",
+    "print('The distance the asteroid would need to move is', dist3,'arcseconds')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 3\n",
+    "As we discussed in class, one of the key backgrounds for gamma-ray telescopes are cosmic rays. Cosmic rays are charged particles—usually protons or electrons but can include atomic nuclei such a alpha particles (helium) or iron. Because of their charge cosmic rays spiral in the magnetic field of the galaxy. From the perspective of the Earth they appear to be coming uniformly from all directions like a high energy gas, and the direction the cosmic ray is travelling when it reaches the Earth tells us nothing about where it came from because we don't know what tortured path it has taken through the galaxy to reach us. However, at trillion electron volt energies and above, the spiral loops are fairly big and the sun and the moon will block cosmic rays. This means the sun and the moon appear as holes in the cosmic ray sky (cosmic rays from that direction are absorbed).\n",
+    "\n",
+    "Assume in a moon sized patch on the sky we normally have a cosmic ray rate of 1 cosmic ray per minute (arrivals are random in time). If we can observe where the moon is for 8 hours per night (not too close to the horizon) and we observe for 15 days and see 6800 cosmic rays, what is the signficance of our moon shadow detection?\n",
+    "\n",
+    "As always:\n",
+    "\n",
+    "What is the question in words?\n",
+    "Translate the question to math, describing both the expected background pdf() and the integral you need to do.\n",
+    "Convert to sigma.\n",
+    "There are some unique aspects to this problem. Make sure you explore them in your explanation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you look in any moon sized patch of sky, you would expect to see an average of 7200 cosmic rays under the conditions described above. If you see less however, it could be significant because it could mean the moon has appeared in that section of the sky for which you are observing. The statistical question: If you look in a moon-sized patch of sky, and you want to determine the amount of space the moon is obscuring in your observation patch, what is the probability you see less than 7200 cosmic rays in the time period described?"
```

Comment:
Make it a bit more clear which part of the question you are answering in these paragraphs